config/manager/manager.yaml

```diff
@@ -49,6 +49,7 @@ spec:
         - "--excluded-namespace=kube-public"
         - "--excluded-namespace=hnc-system"
         - "--excluded-namespace=kube-node-lease"
+        - "--exclude-objects-with-label=cattle.io/creator=norman"
```

Comment:
Maybe add a comment above this line about why this is here - e.g. "preserve preexisting behaviour, please don't add any more to the default list of args"